plugin.video.arteplussept/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.arteplussept" name="Arte +7" version="1.0.1" provider-name="bmf">
+<addon id="plugin.video.arteplussept" name="Arte +7" version="1.0.2" provider-name="bmf">
     <requires>
         <import addon="xbmc.python" version="2.25.0"/>
```

Comment:
2.25.0 is xbmc.python version of Krypton. So this should either be set to 2.14.0 (Gotham version) or submitted to the krypton branch.
With the current situation one would think it is available for Gotham and higher, but it will be only available for Krypton and higher.

Reply:
You can see the download stats for your add-on at https://mirrors.kodi.tv/stats?downloadstats&filter=plugin.video.arteplussept-.

1. submit your add-on with a higher version number to krypton branch or jarvis if you still want to provide updates for that version too.
_Reminder: 1.0.0 and higher already was only available for Krypton because of the xbmc.python version._
2. submit a PR to gotham branch where you mark your add-on broken and do at least a minor version bump.

Tell me if I should merge this PR in the current state or if you already want to drop support old Kodi versions.

Reply:
Thank you for the help !

I'd like to merge this to gotham for now and I will submit 1.1.0 to Jarvis in the future (It should work since I downgraded python version)